README.md

```diff
@@ -61,8 +61,9 @@ impl EventHandler for Handler {}
 
 #[tokio::main]
 async fn main() {
-    let framework = StandardFramework::new().group(&GENERAL_GROUP);
-    framework.configure(|c| c.prefix("~")); // set the bot's prefix to "~"
+    let framework = StandardFramework::new()
+        .group(&GENERAL_GROUP)
+        .configure(|c| c.prefix("~")); // set the bot's prefix to "~"
```

Comment:
```suggestion
        .configure(Configuration::new().prefix("~")); // set the bot's prefix to "~"
```
Now that #2615 has been merged, this should include those changes as well. Be sure to include `Configuration` in the imports as well.